store/sqlstore/file_info_store.go

```diff
@@ -58,8 +58,14 @@ func (fs SqlFileInfoStore) Save(info *model.FileInfo) (*model.FileInfo, error) {
 		return nil, err
 	}
 
-	if err := fs.GetMaster().Insert(info); err != nil {
-		return nil, errors.Wrap(err, "failed to save FileInfo")
+	n, err := fs.GetMaster().Update(info)
```

Comment:
0/5 but I think most of our other `Save` methods only do an insert and fail if you're trying to update something

Reply:
oki. i've moved my stuff into a Upsert method